runtime/ftplugin/sml.vim

```diff
@@ -0,0 +1,29 @@
+" Vim filetype plugin file
+" Language: 		SML
+" Filenames:		*.sml *.sig
+" Maintainer: 		tocariimaa <tocariimaa@firemail.cc>
+" Last Change:		2025-10-03
+
+if exists('b:did_ftplugin')
+  finish
+endif
+let b:did_ftplugin = 1
+
+let s:cpo_save = &cpo
+set cpo&vim
+
+let b:undo_ftplugin = 'setl com< cms< fo<'
+
+setlocal formatoptions+=cqort
+
+if has('comments')
```

Comment:
This test is always true now and just remains for compatibility so it can be removed.  It's also not the appropriate test for `'commenstring'`.

Reply:
this is not yet resolved